int/integration.go

```diff
@@ -0,0 +1,141 @@
+package int
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	configv1 "github.com/openshift/api/config/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/openshift/route-monitor-operator/api/v1alpha1"
+	monitoringopenshiftiov1alpha1 "github.com/openshift/route-monitor-operator/api/v1alpha1"
+	monitoringv1alpha1 "github.com/openshift/route-monitor-operator/api/v1alpha1"
+)
+
+type Integration struct {
+	Client     client.Client
+	clientChan chan struct{}
+	mgr        manager.Manager
+}
+
+func NewIntegration() (*Integration, error) {
+	scheme := runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+
+	utilruntime.Must(monitoringv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(monitoringv1.AddToScheme(scheme))
+	utilruntime.Must(routev1.AddToScheme(scheme))
+	utilruntime.Must(configv1.AddToScheme(scheme))
+
+	utilruntime.Must(monitoringopenshiftiov1alpha1.AddToScheme(scheme))
```

Comment:
this can be used fro
```suggestion
        import ( constinit "github.com/openshift/route-monitor-operator/pkg/consts/test/init" )
	scheme := constinit.Scheme
```

Reply:
in this case I think there are some missing in github.com/openshift/route-monitor-operator/pkg/consts/test/init

Reply:
if they are missing, add them in addition to the base ones? makes it easier to distinguish between duplication and what is required